fs/expose/dokan/__init__.py

```diff
@@ -89,22 +100,46 @@
 import logging
 logger = logging.getLogger("fs.expose.dokan")
 
+if six.PY3:
```

Comment:
Yeah, on another Python project I'm involved with ( https://github.com/RPi-Distro/python-gpiozero ) which supports both Python2 and Python3, the approach taken is to assume Python3 the normal case, and have Python2 as the special case.
(But ISTR there *are* still a few rough spots here and there with PyFilesystem's Python3 support)